src/modules/comments.js

```diff
@@ -0,0 +1,74 @@
+const { default: commentsCounter } = require('./counter/commentsCtr.js');
+
+const appId = JSON.parse(localStorage.getItem('appId'));
+const commentUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments?item_id=`;
+const addCommentUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments`;
+
+const getComments = async (id) => {
+  try {
+    const response = await fetch(commentUrl + id);
+    const data = await response.json();
+    localStorage.setItem(`${id}comments`, JSON.stringify(data));
+  } catch (e) {
+    console.error(e.message);
+  }
+};
+
+const showComments = (id) => {
+  const commentHeadin = document.getElementById(`commentsHeading${id}`);
+  const commentList = document.getElementById(`commentList${id}`);
+  const comments = commentsCounter(id);
+  commentHeadin.innerHTML += ` (${comments.length || 0})`;
+
+  if (Array.isArray(comments)) {
+    comments.forEach((c) => {
+      commentList.innerHTML += `
+                <p>${c.creation_date} ${c.username} : ${c.comment}</p>
+            `;
+    });
+  } else {
+    commentList.innerHTML = '<p>There is no comments yet.</p>';
+  }
+};
+
+const addComment = async ({ id, username, comment }) => {
+  const response = await fetch(addCommentUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+      username,
+      comment,
+    }),
+  });
```

Comment:
- [ ] kindly consider making sure that when someone adds a comment the add comment button POST request to the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) . the data is expected to get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.currently when I add a comment the screen is getting updated but nothing is posted to the API cause once I refresh the page the comment disappeared and the count returns back to zero fixing that will make your comments meet the requirement

Reply:
Fixed.